requirements_dev.txt

```diff
@@ -0,0 +1,13 @@
+# CPU Compatible Requirements
+-r requirements.txt
+
+# Dev:
+# Development, documentation, testing related dependencies.
+pytest
+pytest-ordering
+sphinx
+pypandoc
+sphinx_rtd_theme
+sphinx-jsonschema
```

Comment:
no version number? in the context that you just deleted this has a version number https://github.com/ivadomed/ivadomed/pull/757/files#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552L19

Reply:
Good catch. Yeah, I think this PR started before your PR so this needs to be manually added in this new file. I will added it.